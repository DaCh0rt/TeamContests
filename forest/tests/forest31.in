865 840
-1639 -7387
3950 8529
-8193 -4001
-257 -8279
-4336 5835
-4975 5181
5215 5153
-66 -4286
-2868 -7654
507 -356
784 -665
-4267 5613
4585 3629
3368 9612
-7924 7130
-5806 -6022
-2250 -2252
-7751 -9568
-8048 -6006
3062 -4329
8001 9156
-1318 7853
3694 -6134
7526 4961
8619 -5950
-8539 2763
8983 -6139
-1633 -5346
-5445 900
-742 -3544
-2844 -3273
-246 5875
5738 -4513
1033 3313
3937 -4160
8266 8556
9044 -940
323 -9985
9237 3729
995 6387
-7564 5470
-757 -6774
-4894 -1102
-3342 -5234
-6241 -6240
2805 -5131
-755 2098
5929 9694
-5746 -7338
-3830 4456
1263 749
5994 -6012
-5989 -3208
6124 5135
6260 -4397
-8055 9827
-1965 120
-1480 5686
3232 -7873
5822 6921
7557 1550
8622 -7021
3352 5926
7038 8538
-4867 6412
-436 4373
-700 -974
-2942 9327
-6216 -1492
4254 -5001
-6921 7550
-531 -2666
-2928 -6258
1356 -1413
-7462 2637
2392 4433
1105 -1826
-1801 2801
-236 -6814
9922 -8150
-3098 3351
1052 5786
6796 1088
-6996 -7016
-4136 -5614
-8735 5035
9440 5628
2652 -1748
8153 -4556
-2749 3828
-7323 -9992
-2845 -3704
3367 -5117
2587 -4106
6956 -5808
-4465 -9316
-3179 -3026
2042 -4632
-1947 -1330
1911 4746
-1329 5688
7077 -6498
-3653 9452
-1952 -9756
8359 -7923
8320 -5360
6483 -3012
8296 -3575
-6384 -5750
3194 -2452
6711 -2800
377 -5612
9885 7784
-4371 -4766
25 7217
3370 -8383
4727 -9866
-9896 422
9136 -1001
-1096 -8149
-558 6428
6262 4848
1993 -3676
-8685 -9
7180 -6462
5430 -1027
2740 330
911 -7219
9263 887
-879 -8773
6143 -6511
2056 -5933
7342 -5048
-7918 -6947
-6070 -1318
8526 8539
7564 361
-5697 7807
7547 493
9722 -1167
1830 8956
4587 2482
-2362 7175
-786 -2584
-4892 -8303
-6910 7405
8730 6564
9573 -8289
-6714 2592
5557 3500
9990 9585
-4537 -7561
6617 -7246
-6046 5291
3567 6460
-8313 -5108
-5459 3893
2350 -4335
4842 8871
2421 -9383
-3880 1518
-8244 7010
6749 291
-4347 -4597
7121 -554
7057 625
-9842 941
5565 8226
-5290 695
65 -7496
-1169 3275
-8309 -7743
-8881 -1390
-7357 -7814
-646 -6886
-8994 -893
-835 4573
-9760 -9268
-1684 -3179
7735 -3439
4042 7860
490 -2066
4359 5311
-5130 -2578
-2383 -8404
8698 1378
8677 -1946
4455 7284
6356 -3296
-9679 -9610
-4025 -5412
-5715 -8920
-4035 -4818
-9489 4717
9527 -4027
-6828 -4704
4040 -922
5662 -2893
4934 6159
6740 -887
884 -6228
1151 6444
-7694 6144
-8732 -6447
-3488 -1517
-4370 -6486
1386 6875
-7844 -1387
-6196 1729
6054 2429
-4762 -5642
-6177 -6566
2363 7110
-3859 -3229
2186 -8353
2180 -4348
2375 -7076
-3351 -770
-1080 2324
6537 -9263
6247 -2217
7102 -9171
-5085 -407
6515 -8821
-3975 1869
5034 1082
1968 7731
-9827 4591
4333 -956
4415 -8831
8363 376
-4189 -56
-1356 -4085
9120 -295
6205 -2727
7388 -7434
-8567 9726
-4814 113
-8596 7654
-3587 5860
1641 1692
5779 9523
4511 -8738
4053 1475
-3200 4521
8038 5346
-5399 8359
-8017 -3718
5046 -2875
437 -9254
-8547 442
-6086 5058
-7701 -9717
3747 8904
8802 6156
-7852 -1805
-9246 -9247
-7498 5608
-6066 -175
-3058 5476
7547 4816
7570 4752
-979 -2962
-1170 -779
3203 4071
2873 -977
8907 -7959
-7101 3646
-9232 -4120
6682 5040
3904 8074
7486 9847
-6491 -6082
6021 970
-1769 -3562
-2812 -826
8291 -8550
7073 7079
9022 4134
-4870 2751
3282 1382
3176 6663
-6072 -8261
6383 2707
638 -7682
8547 6970
2987 4574
4663 6238
-7903 8824
6707 -6579
-9943 -1462
4661 3484
2616 7538
-5906 -5881
9811 -4682
-4790 6763
-2973 2660
-8122 185
8191 -5064
-6732 -4049
-4737 6952
-5427 4699
5385 1225
-410 1812
8117 4349
3739 -192
3435 9425
651 5050
-8689 4248
2252 6988
-7594 8634
-6803 -351
-2720 7280
7347 -9584
5526 9116
-3061 -5692
7927 -7485
-2891 2180
8768 -9183
2433 8519
-8713 -6188
1491 6043
474 5960
-660 8387
-659 4642
-1629 2544
-8284 -9822
-3362 -6966
9317 505
-1393 3030
-3060 -9786
7313 -5520
-9701 -274
-3846 -410
5094 3612
1455 -4361
359 9002
8881 -2561
9362 8191
-234 -2484
5489 4553
-1345 3656
1888 4314
7821 -3619
-322 -8502
120 9044
6998 5014
-3943 4221
-2972 845
112 2159
7003 -7638
-9322 -2952
-154 -4836
-7769 667
742 -387
2505 -8920
7086 -2892
1593 2961
-2922 9882
-7006 -8087
6004 -1418
5501 -3625
-2863 -6084
-5232 1013
7121 947
1177 4929
8376 -5401
435 -1453
-5938 3450
8927 3389
-7331 824
-9287 -751
-7699 2817
426 -4515
-4260 -8406
-7909 -3061
6186 3224
-221 -3860
-7031 3449
-2986 1950
3284 5887
-840 8493
-1256 8107
-8338 125
-9434 3111
9430 9642
7564 2703
-9837 -921
2491 -7727
-7453 -5213
-5112 143
1414 -8473
2929 -9952
-7315 145
-7197 -7552
-9537 9876
4675 8132
512 9301
1424 1225
-2962 4204
382 -3484
-5727 7924
-7613 -5897
420 5686
-2221 -2356
7624 9439
-7874 -9352
700 -622
-4668 -8034
6211 -8616
8868 7158
8040 -2459
-8232 367
3071 -9305
-5500 -3258
3744 -5921
5660 -4415
-9752 -1182
1540 9524
-2504 9362
7953 9564
1813 -4554
-137 2409
3789 -271
-4447 6491
-7383 4828
115 1391
-5912 -5573
7656 2836
-3185 6165
9356 -4429
7925 -6504
3170 4177
6303 -6497
1012 -1040
-1731 6205
-4105 411
174 5817
-116 -4556
6685 -8659
6769 -3539
9761 -458
6571 6404
1360 -5004
-962 7709
-4164 -8084
-3172 -9733
-8765 3702
5973 -290
-919 2412
1636 -7724
-5029 -4622
9064 -7214
1202 -9679
-9883 7876
4773 -4582
-343 1299
-2419 4658
1439 -3597
6901 -1854
5714 -9815
2395 -1599
5087 -4273
8249 1511
9345 -8344
6649 -837
3593 -1874
-9292 1749
-9764 8824
5801 8607
6388 -7108
4783 7179
3498 -829
-9081 4514
1522 5238
-4697 -8058
8479 7141
6859 -4584
7706 2222
2217 -5127
174 1803
7254 -6184
265 3326
-5151 5302
-4233 -5779
2227 2421
9507 -47
7998 -9229
-2438 -1979
-5111 5373
1367 -541
9761 -595
-39 278
-6288 5034
9386 6087
-4788 -9474
-2189 7998
-7639 8704
-6403 1195
7029 6397
3882 -6461
2648 1986
6146 9785
7073 -2618
9663 3862
6970 4161
7765 4394
-9531 -959
-1496 3168
-3591 -6077
7487 -3446
5022 -6787
-7165 -208
4921 -7816
-1825 9730
2313 -9987
5724 -286
5701 -8408
2530 9261
-6152 -8660
-655 -587
33 -5270
6247 -8047
9503 -5513
-427 447
340 2993
-3027 -1471
1218 -2066
-1521 8702
-6384 9816
2262 6958
-9561 -1273
7917 -3337
1340 233
-5766 -7042
-633 -5198
-1630 2765
-4453 4607
8011 -6731
5569 -9700
-3308 5462
409 -7073
2226 -1657
5843 -7052
-4701 4520
-6172 3151
-103 2364
-7850 5885
4723 6131
394 -6777
-2407 6582
-8152 7184
4062 3887
-1249 682
-2169 -5081
1879 6788
-4108 5269
5946 4339
8330 5306
7437 2958
4016 -8247
7491 8229
5015 6841
-7787 7223
-8163 -4481
5031 -9895
-4443 -495
-7375 9493
3761 4528
-9148 5346
-104 6021
7300 -6426
-3901 1094
-7483 6205
1427 637
-5450 -6243
9700 -1912
5288 -9125
3247 -5359
-6322 -5432
-4162 9189
3065 1455
3321 -6089
-2951 7382
-2712 -796
-771 7514
-5552 7161
4857 8789
-7107 -4221
-4147 -6084
-7701 6708
9387 4142
3790 464
8894 -9927
-6478 -4840
3734 4464
-9857 -1324
3184 -9222
9814 -7957
7482 5184
-2990 1053
-9895 5719
-5384 -3557
-7920 9916
-6817 6372
-5495 -5433
1432 -1667
-5512 2911
6406 7081
7716 5277
-9745 5058
6931 611
-720 -5157
-4335 -9521
-306 -8533
-3697 5053
-1079 995
2887 416
7998 6833
1005 2336
1331 851
2102 5913
8688 920
2350 -1408
8684 3584
-8588 -7148
7084 -7286
-259 -3440
-3271 6939
-8046 -2524
8503 -6613
-7958 4773
-5695 -9022
6169 1804
-6356 7124
-4860 -18
1777 7740
-3389 -6711
-9892 6891
1650 3139
-117 -6883
-2083 -3671
7827 184
8029 -2708
2858 -3592
3098 5807
3700 -9547
4248 -3828
9405 -7891
3123 4316
2857 2214
5497 -2311
-7214 -8625
5282 -1341
-6675 8872
-1355 3418
-2311 -9942
1336 4465
-8850 -2309
-5778 -6807
7373 -7576
-1676 9720
-3682 -5138
2358 -1810
8478 111
-4345 2104
-3721 1501
-9179 1180
8937 1607
5543 -2276
2402 -2698
2938 -1567
5691 -1096
-6427 3405
7107 5841
-8730 3295
-4072 9473
1461 486
-9028 -2820
7504 8973
9147 5243
4273 -4691
8299 -685
8633 3755
5872 -7757
5040 -3857
9243 8441
8366 -3757
-3659 7012
8804 9957
8425 4864
915 -1929
-5131 2876
1714 5992
-2507 4757
-4526 -385
-2686 -8281
5795 5991
-397 7263
3194 3975
-8990 5670
-8880 -8759
1513 -7621
1853 2979
910 -1620
3417 -5430
-537 -5261
2115 8643
-6249 -6304
8131 -8788
-192 -9263
6545 -5543
2433 -802
-4506 -5798
-6568 -1316
-5289 2329
6929 -3948
-3078 7831
-6160 -1740
-3228 3943
-3333 -5854
6445 4481
7118 -5394
2766 7391
-7883 -9742
3727 8959
6515 -1571
-8065 6853
7680 7825
-5499 8685
-1876 2582
3389 9173
-4159 5228
1026 -7899
-9194 5966
474 2927
-3302 -5144
8838 -2611
-4229 3264
-1525 6755
7552 1332
9153 5442
-6512 7937
-7415 9477
6581 -1062
-1627 -3833
-9699 3580
-4927 -3808
7185 -4399
-8052 -8995
-9324 -6810
5424 -6642
598 7945
-6249 3897
-4969 6559
-6312 760
-2968 -753
3577 -5270
-3545 -750
-6885 4218
-3288 -787
-6603 -1390
6322 7012
4144 -1120
-3542 4400
7945 -4262
-3697 -2306
1083 -738
1793 409
9512 313
1776 -1500
7394 -6568
-769 -622
-2290 7468
-9661 -2019
2341 -1963
-3786 9015
-5329 7250
6071 4967
-9801 -8034
9188 9709
9927 8816
-8763 -4537
-6216 -2670
-4015 -5393
-2996 -6664
7744 -6679
6364 -5084
-9184 -8562
-7704 -3877
-5133 -169
3705 -2998
6341 5115
81 -3672
-2488 -8825
2927 -5652
5750 -1051
51 -5681
7704 -2367
-2355 -5223
-8477 -676
811 8507
3397 1623
-2455 8948
-5528 -7191
657 -2262
-2850 -468
-6292 9454
-5983 5356
-5481 1041
-2647 5110
5685 473
7961 7625
-1713 2473
-5364 1101
4861 2531
-4487 -7008
5176 8682
4626 -1939
2176 -6670
-4872 3317
8764 -7745
-981 3177
4427 8302
-3279 2558
-7939 -6471
-5659 4115
-3056 9143
432 4728
-6531 -549
-6059 7333
-2860 1942
-2622 9730
-152 -9462
1550 9632
-3590 8270
-8745 -155
-9167 -4696
1526 1073
-6234 9457
-5780 -3582
601 469
9024 -6710
7800 -3640
1088 -826
2470 -5946
9713 -4858
-8276 -3841
7959 -3247
3521 8108
3951 -9093
7108 -4442
-9335 8073
-6347 4577
7824 8916
5425 -8582
2745 2426
-9932 5673
1168 -5529
8248 5906
1523 -4010
8942 3389
7324 -482
3719 -4422
-7778 -9487
