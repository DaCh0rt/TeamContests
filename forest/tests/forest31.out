82784.38 460474891.26
